### PR TITLE
fix(ui): notes styles

### LINKED ISSUE #18: Fix notes styles
**Problem**
Our current notes inherited Tabler's card styles:

CleanShot 02 11 2024 at 09-04

**Solution**
Mockups: [Figma](https://www.figma.com/design/kQOH4s5DnfWL0p6o6tlMQz/Draft-%5BATS%5D?node-id=1442-8352&t=YeWiWPdiI26dmnnH-1)

Let's change to our previous notes styles:
- Remove bg-color for note container
- Remove excessive paddings from note container
- Set spacing between elements according to the mockups

Note that these changes should be applied after we update our Text Area, Button and Icon Button components to match Tabler's styles.

### PR DESCRIPTION
**Pull request by wrmk**
Closes #18.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/views/ats/positions/pipeline_tab.html.slim [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ruby:
   status_labels = {}
   mobile_status_labels = {}
-  qualified = params[:pipeline_tab] == "qualified"
```
Comment (wrmk) [inline app/views/ats/positions/pipeline_tab.html.slim:4]: rubocop annoying about unused variable

**Comment by wrmk** [comment]
I did not calculate pixels according to mockup, just make it very similar.
task notes
image

profile notes
image

**Review by ksnitsky — COMMENTED** [review]
`.card-title` has a font size of 16px and a line height of 1.5rem (24px), while it should be 14px and 1.5 (21px)

Current:
image

Should be:
image

---
The thread expand button has a different color on hover and borders when clicked on:

Hover:
image

Click:
image

It also goes away when you add a new reply in an expanded track
image

File: app/assets/stylesheets/src/shared/_profile.scss [hunk 2]
```diff
@@ -28,12 +28,67 @@
...
+      border-top: 0;
+      padding: 0.5rem 0 0 0;
+      text-align: end;
+    }
+  }
+
+  .list-group-item.note {
+    padding: 0;
+    border: 0;
+    margin-top: 0.5rem;
```
[the remark this answers is not in the document]
Comment (wrmk) [reply app/assets/stylesheets/src/shared/_profile.scss:90]: i tried to add margin only to collapsed notes, and it looks weird, margin applied only when notes are visible
do you face with similar problem?

https://github.com/user-attachments/assets/d140286b-82b0-4ed2-ac81-b15a0f8143f1

[the remark this answers is not in the document]
Comment (ksnitsky) [reply app/assets/stylesheets/src/shared/_profile.scss:90]: > i tried to add margin only to collapsed notes, and it looks weird, margin applied only when notes are visible do you face with similar problem?

No
Just don't use margins
Use row-gap

[the remark this answers is not in the document]
Comment (wrmk) [reply app/assets/stylesheets/src/shared/_profile.scss:90]: `row-gap` did not work, found way with `mt-2`

File: app/assets/stylesheets/src/shared/_profile.scss [hunk 3]
```diff
@@ -37,32 +37,36 @@
...
+    .card-footer,
+    .thread-collapse-button,
     .list-group-item.note {
-      &:active,
-      &:hover,
-      &:focus {
-        background-color: var(--tblr-body-bg);
-      }
+      background-color: lightgoldenrodyellow !important;
+      box-shadow: 0px 0px 0px 4px lightgoldenrodyellow !important;
```
Comment (wrmk) [inline app/assets/stylesheets/src/shared/_profile.scss:69]: copied from internal project

**Comment by wrmk** [comment]
will create issue for
image

**Comment by wrmk** [comment]
restored blue bg for target note when we open it via url with anchor

**Review by ksnitsky — APPROVED** [review]

**Pull request merged**